src/core/raps/references.ts

```diff
@@ -57,6 +57,7 @@ export interface RapSwapActionParameters<
   flashbots?: boolean;
   quote: QuoteTypeMap[T];
   address?: Address;
+  isBridge?: boolean;
```

Comment:
I think since `isBridge` doesn't change the function behavior, only metadata, it can be confusing having it in the parameters
can we derive when needed with `isSameAssetInDiffChains(assetToSell, assetToBuy)` or something?

Reply:
👍